### DISCUSSION TITLE
Stranded (CHALLENGE MAP)

### DISCUSSION THREAD

**Discussion by Cybrock9000** [nzp-team/nzportable#791] [Map Releases]
this is my 2nd map the first is in the making:[#810](https://github.com/nzp-team/nzportable/discussions/810)
can you beat my score of 10 on pc, 8 on 3ds ;)
![Screenshot 2024-06-20 001023](https://github.com/nzp-team/nzportable/assets/118949998/0c74d766-18aa-49db-905d-7cce3aaab1ef)
 Map download: [DOWNLOAD](https://archive.org/details/stranded-nzp-map-main)

**Comment by Tcup01** [comment]
The rounds get really hard, if it weren't for the zombies that get stuck, I wouldn't of got far. Round 13 :)

**Reply by Cybrock9000** [reply]
thats a problem idk how to solve. good luck ;)

**Reply by Henrico23** [reply]
I beat ur score
i did cuz ray gun ran out of ammo.end of round 13

**Comment by Jeyphr** [comment]
you appear to be using the ".wa_" file as a waypoint file. That's not the waypoint file, the waypoint file is located in "nzp/data/maps". It should have the same name as the map, but with .way as it's file extension

**Reply by Cybrock9000** [reply]
I didn’t put any waypoints. It’s too small to have any and the zombies can see you from the other side too so.

**Comment by ToonLink0001** [comment]
yo this looks good man, love your work!

**Reply by Cybrock9000** [reply]
Thanks :)

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the content is not listed on archive.org

**Reply by Cybrock9000** [reply]
ok, i will do that

**Reply by Cybrock9000** [reply]
Fixed it (sorry it took so long, I am verry busy)

**Comment by Jeyphr** [comment]
Oh my god I just noticed the typo in the title.

**Reply by Cybrock9000** [reply]
🤫
